internal/service/streamconnection/model_stream_connection.go

```diff
@@ -117,12 +117,24 @@ func NewStreamConnectionUpdateReq(ctx context.Context, plan *TFStreamConnectionM
 	return streamConnection, nil
 }
 
-func NewTFStreamConnection(ctx context.Context, projID, instanceName string, currAuthConfig *types.Object, apiResp *admin.StreamsConnection) (*TFStreamConnectionModel, diag.Diagnostics) {
-	rID := fmt.Sprintf("%s-%s-%s", instanceName, projID, conversion.SafeString(apiResp.Name))
+// determines if the original model was created with instance_name or workspace_name and sets the appropriate field
```

Comment:
```suggestion
// NewTFStreamConnection determines if the original model was created with instance_name or workspace_name and sets the appropriate field.
```

nit: doc standard for funcs is to start with func name, and end all comments with period.